src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunner.java

```diff
@@ -127,14 +127,23 @@ private String createFileForAuxClasspath(SpotBugsTask task) {
           Paths.get(
               task.getProject().getBuildDir().getAbsolutePath(),
               "spotbugs",
-              "spotbugs-auxclasspath");
-      Files.createDirectories(auxClasspathFile.getParent());
-      Files.createFile(auxClasspathFile);
-      Files.write(auxClasspathFile, auxClasspath.getBytes(), StandardOpenOption.TRUNCATE_EXISTING);
-      return auxClasspathFile.normalize().toString();
+              "spotbugs-auxclasspath-for-" + task.getName());
```

Comment:
just feeling: `"auxclasspath", task.getName());` would be good replacement, then the directory hierarchy will be more intuitive like:

```
project/
  - build/
    - spotbugs/
      - auxclasspath/
        - spotbugsMain
        - spotbugsTest
```

Reply:
I like that 👍